ocs_ci/helpers/dr_helpers.py

```
@@ -2613,30 +2613,65 @@ def verify_drpc_placement_deletion(cmd, expected_output_lst):
-        )
+    if initial_last_kubeobject_protection_time:
+        for last_kubeobject_protection_time in TimeoutSampler(
+            (2 * kubeobject_sync_interval * 60),
+            15,
+            drpc_obj.get_last_kubeobject_protection_time,
+        ):
+            if last_kubeobject_protection_time:
+                if (
+                    last_kubeobject_protection_time
+                    != initial_last_kubeobject_protection_time
+                ):
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)